crates/libcgroups/src/v2/io.rs

```diff
@@ -127,10 +127,12 @@ impl Io {
     fn apply(root_path: &Path, blkio: &LinuxBlockIo) -> Result<(), V2IoControllerError> {
         if let Some(weight_device) = blkio.weight_device() {
             for wd in weight_device {
-                common::write_cgroup_file(
-                    root_path.join(CGROUP_BFQ_IO_WEIGHT),
-                    format!("{}:{} {}", wd.major(), wd.minor(), wd.weight().unwrap()),
-                )?;
+                if let Some(weight) = wd.weight() {
+                    common::write_cgroup_file(
+                        root_path.join(CGROUP_BFQ_IO_WEIGHT),
+                        format!("{}:{} {}", wd.major(), wd.minor(), weight),
+                    )?;
+                }
```

Comment:
I have changed this based upon https://github.com/opencontainers/runc/blob/main/libcontainer/cgroups/fs/blkio.go#L29-L33 from runc ; and if the weight is `None` the existing code would just panic. However, I'm not sure if this is the correct approach, or the `None` wight should be defaulted to 0 or reported as error.